examples/plot_anomaly_bench.py

```diff
@@ -1,11 +1,12 @@
 """
-===================================
-Anomaly detection's ROC benchmarks
-===================================
+====================================================================
+Comparison of LocalOutlierFactor and IsolationForest performances on 
+classical anomaly detection datasets using ROC curves
+====================================================================
 Benchmark plots of LocalOutlierFactor (LOF) and IsolationForest (IF) on 
```

Comment:
Sorry I meant changing the sentence "Benchmark plots of LocalOutlierFactor..." not the title...